src/connector/src/sink/big_query.rs

```diff
@@ -13,27 +13,43 @@
 // limitations under the License.
 
 use core::mem;
+use core::time::Duration;
 use std::collections::HashMap;
 use std::sync::Arc;
 
 use anyhow::anyhow;
 use async_trait::async_trait;
 use gcp_bigquery_client::model::query_request::QueryRequest;
-use gcp_bigquery_client::model::table_data_insert_all_request::TableDataInsertAllRequest;
-use gcp_bigquery_client::model::table_data_insert_all_request_rows::TableDataInsertAllRequestRows;
 use gcp_bigquery_client::Client;
+use google_cloud_bigquery::grpc::apiv1::bigquery_client::StreamingWriteClient;
```

Comment:
Why are we using 2 bigquery clients? Can we only use one? (The former `gcp_bigquery_client` is quite old now)